config-service-impl/src/main/java/org/hypertrace/config/service/store/ConfigDocument.java

```diff
@@ -62,21 +64,31 @@
   @JsonProperty(value = CONFIG_FIELD_NAME)
   Value config;
 
+  @JsonProperty(value = CREATION_TIMESTAMP_FIELD_NAME)
+  long creationTimestamp;
```

Comment:
Will we have any backwards compatibility issues? For existing documents, will they just default to have a 0 creation/update timestamp, or would we run into errors?

Reply:
Existing documents will default to have the creation and update timestamp as 0. So, we won't run into errors - it's just that it won't have their actual creation timestamp.

Also added a test for `ConfigDocument` to verify this behaviour^.